README.rst

```diff
@@ -201,6 +201,11 @@ description of the current state of the cluster which is under monitoring.
 Configuration keys
 ==================
 
+``alert_file_dir`` (default ``os.getcwd()``)
```

Comment:
The sort could have been in a different PR or a different commit (just the sort operation, no change). My bad 🙇.
Having them in 016d14db might not be the best. That's why I added the GitHub comments to guide reviewers (there's a sort and there's one key that was missing). If that's not good enough, sure it would be possible to split, rebase interactively (pray), force push, modify the test in `aiven-core` to target the new hash. Tell me if you want this.